index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body class="border">
+    <section class="main">
+    <div class = "row">
+        <div class = "column" >
+            <img class="alignment4" src="https://www.apple.com/ac/structured-data/images/knowledge_graph_logo.png?202110290909" alt="" width=100>    
+        </div>
+        <div class="column" >
+            <h1 class="heading">Epidemic Sound</h1>   
+        </div>
+    </div>
+    <div class = "row">
```

Comment:
I word document it is mention header should  be separated from the other table column by top and bottom borders and you did not used table in the html code and u have used div containers instead In short what they mention in word document is we need two tables(each containing two column cells) one for header and another for list and header should be enclosed between top and bottom border